### PR TITLE
Mailerlite integration

### PR DESCRIPTION
**Pull request by rachmlenig**

### CODE REVIEW

File: configuration/integrations/mailerlite.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: "MailerLite"
+---
+
+The Rownd integration for MailerLite makes it easy to keep your subscriber list in sync. The integration will automatically create a new subscriber or update an existing one in your MailerLite account.
+
+#### Setup
+
+[The MailerLite integration can be enabled from the Rownd dashboard.](https://app.rownd.io)
```
Comment (mhamann) [inline configuration/integrations/mailerlite.mdx:9]: Might be better to just link the `Rownd dashboard` part.

File: configuration/integrations/mailerlite.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+4. Enter a descriptive name and click **Next**.
+
+  ![Create the mailerlite integration](/images/integration_mailerlite_setup.png)
+
+#### Authenticate
+
+Enter your MailerLite API key. You can create or retrieve your API key from your MailerLite account.
+
+1. In your MailerLite account, navigate to the Integrations page.
```
Comment (mhamann) [inline configuration/integrations/mailerlite.mdx:26]: Can we link `MailerLite account` or `Integrations` to the right MailerLite URL where they can find these settings?

File: configuration/integrations/mailerlite.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+2. Under MailerLite API click Use.
```
Comment (mhamann) [inline configuration/integrations/mailerlite.mdx:28]: Let's **bold** specific text that the user should act on, like **Use.**

File: configuration/integrations/mailerlite.mdx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+3. Click Generate new token.
+
+4. Name your new token and copy/paste it into the dialog.
```
Comment (mhamann) [inline configuration/integrations/mailerlite.mdx:32]: The dialog in Rownd, right?

File: configuration/integrations/overview.mdx [hunk 5]
```diff
@@ -26,6 +26,7 @@ Integrations are a beta feature, so you may not see them in your account yet. As
 | Firebase (Authentication)                                                                           | Yes           |
 | HubSpot                                                                                             | No            |
 | Mailchimp                                                                                           | Yes           |
+| [MailerLite](/rownd/guides/configuration/integrations/mailerlite)                                   | Yes           |
```
Comment (mhamann) [inline configuration/integrations/overview.mdx:29]: this link doesn't work. you might just want `./mailerlite`

**Review by mhamann — COMMENTED** [review]
see line comments

File: configuration/integrations/overview.mdx (hunk 5 above)
Comment (rachmlenig) [reply configuration/integrations/overview.mdx:29]: All of the links weren't working, so I fixed them!

File: configuration/integrations/mailerlite.mdx (hunk 2 above)
Comment (rachmlenig) [reply configuration/integrations/mailerlite.mdx]: I updated "Integrations" with the correct MailerLite link

File: configuration/integrations/mailerlite.mdx (hunk 4 above)
Comment (rachmlenig) [reply configuration/integrations/mailerlite.mdx]: Yep, I made it clearer.

**Review by mhamann — APPROVED** [review]

**Pull request merged**